assets/js/components/billing/DataCreditsIndex.jsx

```diff
@@ -388,6 +392,20 @@ class DataCreditsIndex extends Component {
                       >
                         Purchase Data Credits
                       </Button>
+                      {
+                        organization.survey_token_sent_at && !organization.survey_token_used && (
+                          <Button
+                            type="primary"
+                            onClick={() => this.openModal("showRedeemSurveyTokenModal")}
+                            style={{
+                              borderRadius: 4,
+                              display: !process.env.SELF_HOSTED ? "inline" : "none"
```

Comment:
we should never enter here if `process.env.SELF_HOSTED` right?

Reply:
yep, vestige